app/src/main/res/values/strings.xml

```diff
@@ -117,24 +117,27 @@
         like better. With enough comparison picks, the app will create a ranking from whom you like
         the most to least.\n
         \n
-        Next the app limits every user to maximum 10 matches. You get matched based on whom you
+        The app limits every user to maximum 10 matches. You get matched based on whom you
```

Comment:
"You get matched based on whom you matched up unless either of you runs out of matches."

That seems like circular logic: "matched based on whom you match up."

Also logically, the structure should probably be:
- Create stack ranking from comparisons.
- Match best mutual choices based on stack ranking.
- Stop matching after 10 matches.

(Right now it's: Rank, limit, then match.